### ISSUE TITLE
What is the meaning for "Exp Name FULL" ?

### ISSUE THREAD

**Issue by HuangLian126** [kris-singh/Deep-Unsupervised-Saliency-Detection#1]
Thanks for your implmentation ! Recently, I am interested in the unsupervised learning, this implmentation can help me. Here, I don not know the meaning of "Exp Name FULL". Can you explain the Full Training?

**Comment by Patrickctyyx** [comment]
> Method for training deep unsupervised sailency detection model.
    Training module implements 4 training methodology
    1. Real: The backbone network is trainined with Ground Truth object, loss = mean((pred_y_i, y_gt_i))
    3. Noise: The backbone network is trained using losses on all the labels of unsupersived methods, loss = mean((pred_y_i, unup_y_i_m))
    3. Avg: The backbone network is trained  using avg of the all the unsupervised methods as ground truth, loss = mean((pred_y_i, mean(unup_y_i_m)))
    4. Full: The backbone network as well as the noise module is trained, the training proceeds as follows

You can refer to https://github.com/kris-singh/Deep-Unsupervised-Saliency-Detection/blob/master/src/deep_unsup_sd.py for more details.

**Comment by HuangLian126** [comment]
@Patrickctyyx Thanks for your reply. I'm training, hhh.
